freeipa.spec.in

```diff
@@ -1121,6 +1121,7 @@ fi
 %{_usr}/share/ipa/*.ldif
 %{_usr}/share/ipa/*.uldif
 %{_usr}/share/ipa/*.template
+%{_usr}/share/ipa/bind.ipa-ext.conf
```

Comment:
Ideally, each file/directory should have an owner (package).
Please, add a custom config:
`%ghost %attr(0640,root,name) %config(noreplace) %{_sysconfdir}/named/ipa-ext.conf`